src/Worker.Extensions.DurableTask/ActivityInputConverter.cs

```diff
@@ -25,6 +25,13 @@ public ValueTask<ConversionResult> ConvertAsync(ConverterContext context)
             throw new ArgumentNullException(nameof(context));
         }
 
+        // Special handling for FunctionContext
+        // This addresses cases where the activity function has only FunctionContext as a parameter.
+        if (context.TargetType == typeof(FunctionContext))
+        {
+            return new(ConversionResult.Success(context.FunctionContext));
```

Comment:
Can you test with returning `ConversionResult.Unhandled();`?

Reply:
Thanks for reviewing! I tested it, and it will throw a error showing that there is an unhandled exception.